### PR TITLE
new term: obstetric disorder

### LINKED ISSUE #3161: New term class: Obstetric disorder
From Workshop: 
Discussions: 
- These terms won’t be in Harrison
- We should have a term for obstetric, which would cover pregnant and post-partum/puerperal state
Note: should not be called reproductive disorder, and ‘reproductive disorder should NOT be a synonym as reproductive disorders are completely different and affect the reproductive system
**AI: Create New term class: Obstetric disorder**
Subclass: 
> pregnancy disorder
> puerperal disorder

related #2741

### PR DESCRIPTION
**Pull request by sabrinatoro**
fix #3161

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -424775,7 +424776,8 @@ synonym: "puerperal disorder" EXACT [MESH:D011644]
 xref: MESH:D011644 {source="MONDO:equivalentTo"}
 xref: SCTID:362973001 {source="MONDO:equivalentTo"}
 xref: UMLS:C0034040 {source="MONDO:equivalentTo"}
-is_a: MONDO:0000001 {source="https://github.com/monarch-initiative/mondo/issues/652"} ! disease or disorder
+is_a: MONDO:0700003 ! obstetric disorder
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:424779]: could you please add a source annotation here
you can add the ticket https://github.com/monarch-initiative/mondo/issues/3161 and your ORCID

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -407448,9 +407448,10 @@ xref: MESH:D011248 {source="MONDO:equivalentTo"}
 xref: NCIT:C35169 {source="MONDO:equivalentTo"}
 xref: SCTID:173300003 {source="MONDO:equivalentTo"}
 xref: UMLS:C0151864 {source="MONDO:notFoundInDiseaseSubset", source="NCIT:C35169"}
-is_a: MONDO:0000001 {source="https://github.com/monarch-initiative/mondo/issues/652"} ! disease or disorder
+is_a: MONDO:0700003 ! obstetric disorder
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:407451]: could you please add a source annotation here
you can add the ticket https://github.com/monarch-initiative/mondo/issues/3161 and your ORCID

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -437627,6 +437629,13 @@ synonym: "small lungs" RELATED [PMID:24836585]
 is_a: MONDO:0005275 {source="PMID:24836585", source="https://orcid.org/0000-0003-0113-912X"} ! lung disease
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700003
+name: obstetric disorder
+def: "Disorder associated with pregnancy, childbirth, and puerperium." [https://orcid.org/0000-0002-4142-7153]
+is_a: MONDO:0000001 ! disease or disorder
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:437636]: please add an equiv axiom (Equivalent To)

'pregnancy disorder' or 'puerperal disorder'

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]

**Comment by nicolevasilevsky** [comment]
this looks great, one minor comment

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:437636]: done

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**